### PR TITLE
[ref] Migrating the TeamPage from Chakra UI to Tailwind.

### LINKED ISSUE #215: [ref] Migrate `Team Page`

### PR DESCRIPTION
**Pull request by Joel-leal**
Descrição

This task is refactoring the TeamPage for styles in Tailwind

Changes

- ref: I replaced the use of Chakra UI components with a more concise and simplified approach, using basic HTML tags and styling with the Tailwind library.

## Preview
Screenshot 2024-10-18 at 14 20 36

Screenshot 2024-10-18 at 14 22 02

> 

Closes #215

### CODE REVIEW

File: packages/components/TeamPage.tsx [hunk 1]
```diff
@@ -13,38 +11,28 @@ type Props = {
...
-          <Grid
-            templateColumns={{
-              base: "1fr",
-              sm: "1fr 1fr",
-              md: "repeat(3, 1fr)",
-              lg: "repeat(4, 1fr)",
-            }}
-            gap={{ base: 4, lg: 6 }}
-          >
+          <div className="grid p-4 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4 gap-4 lg:gap-6">
```
Comment (marco-souza) [inline packages/components/TeamPage.tsx:29]: Although it's a minor detail, I prefer to organize the classes by placing the base classes first and the responsive classes afterward.

```suggestion
          <div className="grid p-4 gap-4 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-4 lg:gap-6">
```

File: packages/components/TeamPage.tsx [hunk 2]
```diff
@@ -13,38 +11,28 @@ type Props = {
 export default function TeamPage({ members }: Props) {
   return (
     <Section className="py-40">
-      <Grid gap={10}>
-        <Heading fontWeight={600} fontSize={{ base: "3xl", sm: "4xl" }} lineHeight="110%" textAlign="center">
+      <div className="grid gap-10">
+        <h2 className="font-semibold text-4xl sm:font-xl text-center leading-8">
```
Comment (marco-souza) [inline packages/components/TeamPage.tsx:15]: This is the page's main title, so it must be an `h1` for correct syntax.

```suggestion
        <h1 className="font-semibold text-4xl sm:font-xl text-center leading-8">
```

Comment (Joel-leal) [reply packages/components/TeamPage.tsx]: change here [c4898bf](https://github.com/podcodar/webapp/pull/216/commits/c4898bfcda80c889a0429fe4e2b1db38301d82d1)

File: packages/components/TeamPage.tsx (hunk 1 above)
Comment (Joel-leal) [reply packages/components/TeamPage.tsx]: change here [febf5b3](https://github.com/podcodar/webapp/pull/216/commits/febf5b3979e913e679cb047a7b55d8d71014e1db)

**Review by marco-souza — APPROVED** [review]

**Pull request merged**